pr_testing/run-pr-profiling.sh

```diff
@@ -95,6 +95,8 @@ for PROFILING_WORKFLOW in $WORKFLOWS;do
     ls -l $LOCALREL/profiling/${CMSSW_VERSION}/${SCRAM_ARCH}/${PROFILING_WORKFLOW}/${UPLOAD_UNIQ_ID}/$BASENAME || true
     AMP="&"
     echo "<li><a href=\"https://cmssdt.cern.ch/circles/web/piechart.php?data_name=profiling${AMP}filter=${CMSSW_VERSION}${AMP}local=false${AMP}dataset=${CMSSW_VERSION}/${SCRAM_ARCH}/${PROFILING_WORKFLOW}/${UPLOAD_UNIQ_ID}/${BASENAME//.json/}${AMP}resource=time_thread${AMP}colours=default${AMP}groups=reco_PhaseII${AMP}threshold=0\">$BASENAME</a></li>" >> $WORKSPACE/upload/profiling/index-$PROFILING_WORKFLOW.html
+    get_jenkins_artifacts profiling/${CMSSW_VERSION}/${SCRAM_ARCH}/${PROFILING_WORKFLOW}/$f ${CMSSW_VERSION}_$f
+    $CMS_BOT_DIR/pr_testing/resources-diff.py $f ${CMSSW_VERSION}_$f > ${CMSSW_VERSION}_diff_$f 2>$f.log
```

Comment:
@gartung , may be add `|| true` to avoid killing the profiiling job ?

Reply:
Done